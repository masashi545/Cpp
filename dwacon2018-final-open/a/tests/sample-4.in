12 34 56
10000 155
