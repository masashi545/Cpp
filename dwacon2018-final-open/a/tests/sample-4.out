610149 610209
